src/components/custom-coverage-date/custom-coverage-date.js

```diff
@@ -0,0 +1,170 @@
+import React, { Component } from 'react';
+import { Field, reduxForm } from 'redux-form';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+import Datepicker from '@folio/stripes-components/lib/Datepicker';
+import IconButton from '@folio/stripes-components/lib/IconButton';
+import Button from '@folio/stripes-components/lib/Button';
+import styles from './custom-coverage-date.css';
+
+class CustomCoverageDate extends Component {
+  static propTypes = {
+    initialValues: PropTypes.shape({
+      beginCoverage: PropTypes.string,
+      endCoverage: PropTypes.string
+    }).isRequired,
+    handleSubmit: PropTypes.func.isRequired,
```

Comment:
I only see two props passed into `CustomCoverageDate` at https://github.com/folio-org/ui-eholdings/pull/151/files#diff-0d953e650d58ec07823a4b3dc2a91366R8; do these other things need to be props?

Reply:
These are passed by Redux form implicitied in the background